### ISSUE TITLE
kallisto quant synthax error when trying to generate log file for multiqc

### ISSUE THREAD

**Issue by dscdscdscs** [pachterlab/kallisto#290]
When I try to run the following in RStudio, I get the subsequent error:

kallisto quant -i file.index -o test -t 4 sample_1.fastq.gz sample_2.fastq.gz &> sample.log
The syntax of the command is incorrect.

However, if I remove "&> sample.log", it will run just fine but will of course not provide me with a log file to use with MultiQC

Any ideas why this might be happening?

**Comment by kascchmc** [comment]
Did you ever find a solution to this? I'm also having the same problem

**Comment by oelakad** [comment]
Hi, did you find a solution for this issue?

**Comment by kascchmc** [comment]
> Did you ever find a solution to this? I'm also having the same problem

Yes, Kallisto doesn't run on Windows. I ran it through at Linux subsystem and everything worked.

**Comment by kascchmc** [comment]
> > Did you ever find a solution to this? I'm also having the same problem
> 
> Yes, Kallisto doesn't run on Windows. I ran it through at Linux subsystem and everything worked.

Yes, Kallisto doesn't run on Windows. I ran it through at Linux subsystem and everything worked.

**Comment by Yenaled** [comment]
^kallisto runs on windows and through RStudio, but the issue is probably the &> at the end of the command. I can't really fix this on the kallisto end as this is not a kallisto-specific issue (&> is a linux/unix redirection thing, and multiqc seems to require you to make a file through this).

**Comment by oelakad** [comment]
> ^kallisto runs on windows and through RStudio, but the issue is probably the &> at the end of the command. I can't really fix this on the kallisto end as this is not a kallisto-specific issue (&> is a linux/unix redirection thing, and multiqc seems to require you to make a file through this).

I think so, as I faced the same thing running it on mac. is there away to get around this on mac?

**Comment by Yenaled** [comment]
^It works on the Mac command-line terminal. Not sure about RStudio though (I don't usually run things on it)

**Comment by oelakad** [comment]
it worked yes (was just slow and did not show any responce till finished, which brought the confusion), thanks much.
